com.unity.shadergraph/Editor/Generation/Templates/BuildVertexDescriptionInputs.template.hlsl

```diff
@@ -3,28 +3,72 @@ VertexDescriptionInputs BuildVertexDescriptionInputs(Attributes input)
     VertexDescriptionInputs output;
     ZERO_INITIALIZE(VertexDescriptionInputs, output);
 
+#ifdef TERRAIN_ENABLED
```

Comment:
Same as above:

    This code has not been tested for URP yet. I copied it in directly from HDRP. This is because HDRP had already streamlined Terrain's TBN calculations, while URP more closely replicates the original builtin terrain code, which has odds and ends scattered all over the place. I haven't yet compared this file with HDRP's Vertex.template.hlsl to see whether the expectations for the output are different yet.

Reply:
Fixed!